python/pyarrow/tests/test_udf.py

```diff
@@ -283,7 +283,7 @@ def test_scalar_udf_array_binary(binary_func_fixture):
                           )
 
 
-def test_scalar_udf_array_ternary(ternary_func_fixture):
```

Comment:
I see where you might have thought instead of `ScalarUdf` to go with `Udf` prefix. I am just curious, how is tabular function different from a regular scalar function in terms of usage and anatomy? Would appreciate the idea in your workds. I have sort of an idea compiled from your code, but would like to know more.

Reply:
See [this post](https://github.com/apache/arrow/pull/14682/#discussion_r1050597587) about similarity and difference.